web/src/lib/components/pages/record.tsx

```diff
@@ -117,6 +124,19 @@ export default class RecordPage extends Component<RecordProps, RecordState> {
 
     this.props.onRecordStop && this.props.onRecordStop();
 
+    let error = this.checkRecording();
```

Comment:
not sure if there are any stated style rules for this project yet, but if you don't (need to) re-assign a variable, using `const` is nicer :)

(same goes for the other variables you introduced or touched)